### PR TITLE
buttons delete me, add me to event

### PR DESCRIPTION
**Pull request by kasiaekiert**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -44,6 +44,7 @@ gem 'bootstrap', '~> 4.3.1'
 gem 'jquery-rails'
 
 gem 'rails-i18n', '~> 5.1', '>= 5.1.3'
+gem 'rubocop-performance', '~> 0.0.1'
```
Comment (womanonrails) [inline Gemfile]: Why did you move this gem here?

**Review by womanonrails — CHANGES_REQUESTED** [review]

**Review by womanonrails — CHANGES_REQUESTED** [review]
You should split this MR into 2 MRs. One for button styling and one for adding this gem (remember to generate `.rubocop.yml` file)

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -2,10 +2,10 @@ module ApplicationHelper
   def flash_message
     flash.map do |key, value|
       level_class = case key
-        when 'notice' then "alert alert-info"
-        when 'success' then "alert alert-success"
-        when 'error' then "alert alert-danger"
-        when 'alert' then "alert alert-warning"
+        when 'notice' then 'alert alert-info'
```
Comment (womanonrails) [inline app/helpers/application_helper.rb]: Here you have the indentation problem.

File: app/models/place.rb [hunk 3]
```diff
@@ -2,7 +2,6 @@ class Place < ApplicationRecord
   has_many :events
   
   def full_adress
-    [name, "-", address].join(" ")
+    [name, '-', address].join('')
```
Comment (womanonrails) [inline app/models/place.rb]: Here you miss space inside the quotes. You should have `' '`

File: config/initializers/devise.rb [hunk 4]
```diff
@@ -60,8 +62,10 @@
   config.strip_whitespace_keys = [:email]
 
   # Tell if authentication through request.params is enabled. True by default.
-  # It can be set to an array that will enable params authentication only for the
-  # given strategies, for example, `config.params_authenticatable = [:database]` will
+  # It can be set to an array that will enable params
```
Comment (womanonrails) [inline config/initializers/devise.rb]: It should be possible to set in you editor command to remove all not needed spaces. Then this message from hound should disappear.

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (womanonrails) [reply app/helpers/application_helper.rb]: You can run rubocop locally it should show similar warnings.

File: config/initializers/devise.rb [hunk 5]
```diff
@@ -44,14 +44,16 @@
 
   # Configure parameters from the request object used for authentication. Each entry
   # given should be a request method and it will automatically be passed to the
-  # find_for_authentication method and considered in your model lookup. For instance,
-  # if you set :request_keys to [:subdomain], :subdomain will be used on authentication.
+  # find_for_authentication method and considered in your model lookup.
+  # For instance, if you set :request_keys to [:subdomain],
```
Comment (womanonrails) [inline config/initializers/devise.rb]: Why you change this comment? This are default comments. You don't need to change them.

Comment (womanonrails) [reply config/initializers/devise.rb]: Could you move back changes in this file?

File: config/initializers/wrap_parameters.rb [hunk 6]
```diff
@@ -3,7 +3,8 @@
 # This file contains settings for ActionController::ParamsWrapper which
 # is enabled by default.
 
-# Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
+# Enable parameter wrapping for JSON. You can disable this
```
Comment (womanonrails) [inline config/initializers/wrap_parameters.rb]: Move back change in comment.

**Review by womanonrails — CHANGES_REQUESTED** [review]

File: config/initializers/devise.rb (hunk 5 above)
Comment (kasiaekiert) [reply config/initializers/devise.rb]: yes, I can, do it right now. 

rubocop show it as too long

File: config/initializers/wrap_parameters.rb [hunk 7]
```diff
@@ -1,7 +1,6 @@
 # Be sure to restart your server when you modify this file.
 
-# This file contains settings for ActionController::ParamsWrapper which
-# is enabled by default.
+# This file contains settings for ActionController::ParamsWrapper which is enabled by default.
```
[the remark this answers is not in the document]
Comment (kasiaekiert) [reply config/initializers/wrap_parameters.rb:3]: Should i worry about it and change? @womanonrails

**Comment by kasiaekiert** [comment]
Can I do merge pull request? 

I add extension to show whitespaces - will be easier to work with these problems

File: config/initializers/wrap_parameters.rb (hunk 7 above)
[the remark this answers is not in the document]
Comment (womanonrails) [reply config/initializers/wrap_parameters.rb:3]: You should not change default comments if you don't need to.

File: config/initializers/devise.rb [hunk 8]
```diff
@@ -44,14 +44,16 @@
 
   # Configure parameters from the request object used for authentication. Each entry
   # given should be a request method and it will automatically be passed to the
-  # find_for_authentication method and considered in your model lookup. For instance,
-  # if you set :request_keys to [:subdomain], :subdomain will be used on authentication.
+  # find_for_authentication method and considered in your model lookup.
```
Comment (womanonrails) [inline config/initializers/devise.rb:47]: Please do not change comments form external gems if you don't need to. This change no make sense.

**Review by womanonrails — CHANGES_REQUESTED** [review]

**Review by womanonrails — APPROVED** [review]

**Pull request merged**